### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by hazwanizafirah**
ready

### CODE REVIEW

**Comment by hazwanizafirah** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```
Comment (thaodtd-0665) [inline app/views/layouts/application.html.erb:4]: Research and apply "i18n" into project for translate string (using display on UI for end user)
https://guides.rubyonrails.org/i18n.html#providing-translations-for-internationalized-strings

```suggestion
    <title>t("title")</title>
```
```
# config/locales/en.yml
en:
  title: Sample App
```

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (thaodtd-0665) [inline config/database.yml:1]: 1. Config this file into .gitignore then run bellow command line --> do not push this file to pull request
`git rm --cached config/database.yml `
2. Create a new file with name "config/database.yml.example" --> this file using push to pull request

**Comment by thaodtd-0665** [comment]
Your pull have 2 commit, recommend rebase 1 commit/pull request
https://stackoverflow.com/questions/2563632/how-can-i-merge-two-commits-into-one-if-i-already-started-rebase

File: config/database.yml (hunk 2 above)
Comment (hazwanizafirah) [reply config/database.yml:1]: can you help me how to resolve this problem? I do not where to start to solve this problem.

Comment (hazwanizafirah) [reply config/database.yml:1]: I run this command and it appear like this
$ git add config/database.yml.example
fatal: pathspec 'config/database.yml.example' did not match any files

**Comment by hazwanizafirah** [comment]
ready

File: config/database.yml (hunk 2 above)
Comment (huongnt-2545) [reply config/database.yml:1]: Steps:
- You need to create a new file into `config/` folder which named `database.yml.example` with the same format database.yml file
- gitignore `config/database.yml` file

**Review by huongnt-2545 — COMMENTED** [review]
Your pull request has 2 commit, you need to combine in 1
Have comment not fix

File: config/credentials.yml.enc [hunk 3]
```diff
@@ -0,0 +1 @@
+21PtXuVMyt8lb8RHrgWfBANuHwT1WksPtcgHD/PVAxFjvo4zEhuvpggH4z/C+2pju/ezmozHN4TuiR3ajZbMj40mcndiqzfNv6FS1z7PVE3fhO2A0HISz2Lmc3+4vWJ3jIRhKxH1E7RAiAAwdVdWt6Q6e/y9nbEGLhYhEDfGHm18OS8Qb2hfBYk1BgQ3R91DLSqtSYPvWBQ5lzac9ZaLXo/35nwjul2bfPiXLg3y9EFSnO0PxGFCCLk95snumbtyvfesTZz3atOusm9hBv3AsP6VeRee73vxKWWB8CW4micMzTBnDmcbLT5hR/a6fiy8sDd56C3nsrzZ76vHzxVHrHvvoueZ2cB4kaR+hY/MvPBaWfJVLeWsnquU6XonhQiEtiNEoW0rsyWcLMUKTR4UjIjLBHO4fvwUsupY--cqelOPAJFlA5f618--vKVaYQY+Moh33ep/U0FR+Q==
```
[the remark this answers is not in the document]
Comment (hazwanizafirah) [reply config/credentials.yml.enc:1]: done

**Pull request closed without merging**